src/components/permissions/rowLevelPolicyHelpers.ts

```diff
@@ -119,11 +119,55 @@ const generateRowLevelPolicies = (permissionRows: permissionPolicyColumns[]) =>
   return policies
 }
 
+export const modifyValueInObject = (
+  obj: object,
+  matchFn: (key: string, value: object) => boolean,
+  modifyFn: (value: object) => string
+): object => {
+  if (typeof obj != 'object') {
+    return obj
+  }
+  if (Array.isArray(obj)) {
+    console.warn('Arrays are not supported')
+    return obj
+  }
+
+  return Object.entries(obj).reduce(
+    (acc, [key, value]) => ({
+      ...acc,
+      [key]: matchFn(key, value) ? modifyFn(value) : modifyValueInObject(value, matchFn, modifyFn),
+    }),
+    {} as object
+  )
+}
+
+const updateRulesUseViewsInsteadOfTables = (rules: object) => {
+  const tablesToTurnIntoViews: { [key: string]: boolean } = {}
+
+  return {
+    rules: modifyValueInObject(
+      rules,
+      (key, value) => key == '$from' && typeof value == 'string',
```

Comment:
As discussed, can you add  `&& !(value.startsWith('private.'))` qualifier too?

Reply:
2b09c25e54d6929bca01b80f2cfb6d41a0a3bca1